app/helpers/barnardos/action_view/form_helpers.rb

```diff
@@ -118,6 +118,12 @@ def checkbox_group_vertical(name, legend, selection_list, values = [], error: ni
                 end
               )
             end
+            concat(
+              content_tag(:div,
+                          "#{name.to_s.humanize} #{error[0]}",
```

Comment:
The variable name says "error" but this is still an array. Can it just be a string?